### DISCUSSION TITLE
FX protocol unselectable

### DISCUSSION THREAD

**Discussion by Lukemiester16** [pascallanger/DIY-Multiprotocol-TX-Module#1036] [Q&A]
New to GitHub and not super fluent with this stuff, so let me know if this question should be elsewhere.
I just purchased a QUDI 560 (Maule M7) and I am trying to bind it to my TBS MPM on my FrSky Tandem X18. I know it's in the FX protocol, but I can't even select the FX protocol. tapping or clicking makes no difference, it just ends up on the FlySky AFHDS 2A protocol, multiple lines above it. I currently have version 1.3.4.0. what's going on?

**Comment by pascallanger** [comment]
If you've installed the latest test AIR firmware on your module from here https://downloads.multi-module.org/latest-test/ and still can't select the FX protocol then you need to ask FrSky since it has nothing to do with Multi.
